### DISCUSSION TITLE
PyHost 3.0

### DISCUSSION THREAD

**Discussion by Ames-hub** [Ames-hub/pyhoster#10] [General]
Added FTPS
Added Warden
Added UserMan
Probs many other things I forgot about.

FTPS
A standard, pyftpdlib, File Transfer Protocal server integrated to work with Pyhost and all your projects.
See more at ./technical_notes/ftp.md

Warden
A Lockout system to protect pages by adding a Pin code to them that must be entered before access. 
See more at ./technical_notes/warden.md

User Manager (UserMan)
A User management system integrated with FTPS, and in the future extra features. (such as a planned WebGUI)

Roadmap. Priority ascending (added first at top, last at bottom)
Fully integrated API
WebGUI
Auto Updater
Plugin Support
Backups over Cloud storage

Roadmap not guaranteed to be followed.
Enjoy the update, and please report any bugs/things that shouldn't be to the Issue tracker or me directly.

<hr /><em>This discussion was created from the release <a href='https://github.com/Ames-hub/pyhoster/releases/tag/'>PyHost 3.0</a>.</em>

**Comment by Ames-hub** [comment]
beeg changes

**Comment by SniffBakaSniff** [comment]
:O so fancy
